covalent_dispatcher/_cli/service.py

```diff
@@ -304,6 +304,7 @@ def start(
         {
             "user_interface.address": "0.0.0.0",
             "user_interface.port": port,
+            "develop": develop,
```

Comment:
@FyzHsn @wjcunningham7 This one line is breaking all the tests. I can not figure out why. Any idea how to set this flag correctly?

Reply:
Hmmm It's also breaking the cli tools! Lemme see how we can get around it

Reply:
Yeah I know why this is breaking - there was a different reason to include the develop flag for CLI. Unfortunately, this is creating a clash. I will work on getting this working